composer.json

```diff
@@ -10,11 +10,14 @@
         }
     ],
     "require": {
-        "php": "^7.3|^7.4|^8.0|^8.1"
+        "php": "~7.4.0 || ~8.0.0 || ~8.1.0 || ~8.2.0"
```

Comment:
I guess this was the initial intention. 

before PHP 8.2 compatibility was implicitly given, because `^8.0` and `^8.1` include 8.2

Reply:
I'd prefer to go with a more lax contraint here:

```json
"php": "^7.3 || ^8.0"
```

The implementation is so trivial and should not really break in any upcoming 8.* release.

I don't want to be responsible in holding back anyone upgrading to the ne 8.* release because this lib has a unnecessary strict constraint and did not bump the version up in time.

Reply:
I'd prefer to go with a more lax contraint here:

```json
"php": "^7.3 || ^8.0"
```

The implementation is so trivial and should not really break in any upcoming 8.* release.

I don't want to be responsible in holding back anyone upgrading to the ne 8.* release because this lib has a unnecessary strict constraint and did not bump the version up in time.

Reply:
totally understandable and reasonable :+1: 

will fix that